docs/admin.rst

```diff
@@ -187,6 +197,24 @@ possession when they attempt to log in to SecureDrop.
 .. |Verify YubiKey|
   image:: images/manual/screenshots/journalist-admin_new_user_two_factor_hotp.png
 
+.. _Updating the Workstations:
+
+Updating SecureDrop Workstations
+--------------------------------
+
+In addition to Tails updates, once you have upgraded to SecureDrop 0.6 or later,
+you can keep the SecureDrop software up to date on each *Journalist Workstation*
+and *Admin Workstation* using the ``securedrop-admin`` CLI. In each workstation:
+
+.. code:: sh
+
+  cd ~/Persistent/securedrop
+  ./securedrop-admin setup
+  ./securedrop-admin update
+
```

Comment:
Are there cases where we would have to run `./securedrop admin setup` after pulling the latest tag? Or do you think this be in release notes on an as-needed basis?

Reply:
This... is a good point. I'll add `securedrop-admin setup`, since if there are _not_ changes to the requirements then it doesn't hurt.